### PR TITLE
Remove the new keyword from all recipes

### PR DESCRIPTION
**Pull request by brianegan**
This PR removes the `new` Keyword in all recipes, making use of that Dart 2 🔥

### CODE REVIEW

**Comment by brianegan** [comment]
Heya @mjohnsullivan -- another ping :) Would appreciate a review on this one as well when ya get a chance!

File: cookbook/navigation/returning-data.md [hunk 1]
```diff
@@ -170,13 +170,11 @@ update the `_navigateAndDisplaySelection` method in our `SelectionButton`.
...
     context,
-    new MaterialPageRoute(builder: (context) => new SelectionScreen()),
+    MaterialPageRoute(builder: (context) => SelectionScreen()),
   );
 
   // After the Selection Screen returns a result, show it in a Snackbar!
-  Scaffold
-      .of(context)
-      .showSnackBar(new SnackBar(content: new Text("$result")));
+  Scaffold.of(context).showSnackBar(SnackBar(content: Text("$result")));
```
Comment (Sfshaza) [inline cookbook/navigation/returning-data.md]: Why is this formatted differently? Did you run it through the formatter?

File: cookbook/navigation/returning-data.md [hunk 2]
```diff
@@ -222,45 +220,43 @@ class SelectionButton extends StatelessWidget {
...
       context,
-      new MaterialPageRoute(builder: (context) => new SelectionScreen()),
+      MaterialPageRoute(builder: (context) => SelectionScreen()),
     );
 
     // After the Selection Screen returns a result, show it in a Snackbar!
-    Scaffold
-        .of(context)
-        .showSnackBar(new SnackBar(content: new Text("$result")));
+    Scaffold.of(context).showSnackBar(SnackBar(content: Text("$result")));
```
Comment (Sfshaza) [inline cookbook/navigation/returning-data.md]: Same question.

File: cookbook/navigation/returning-data.md (hunk 1 above)
Comment (brianegan) [reply cookbook/navigation/returning-data.md]: Yep! That was caught by the formatter on TravisCI and needed to change it up so the build would pass.

File: cookbook/navigation/returning-data.md (hunk 2 above)
Comment (brianegan) [reply cookbook/navigation/returning-data.md]: Yep, formatter on Travis failed on this one as well.

**Comment by brianegan** [comment]
Thanks for the help @Sfshaza 🎉 👏

**Comment by Sfshaza** [comment]
SGTM, @brianegan! I would merge it myself, but I'll leave it to you.

**Pull request merged**